plugins/modules/lambda.py

```diff
@@ -142,6 +142,13 @@
     type: list
     elements: dict
     version_added: 5.5.0
+  image_uri:
+    description:
+      - The Amazon ECR URI of the image to use.
+      - Required (alternative to runtime zip_file and s3_bucket) when creating a function.
+      - Required when I(state=present).
+    type: str
+    version_added: 6.5.2
```

Comment:
```suggestion
    version_added: 7.1.0
```

6.x is now bug/security fixes only, we can add this to 7.1 without a problem.